### DISCUSSION TITLE
MR60BHA2 mmWave Sensor with Home Assistant | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1788] [Q&A]
# MR60BHA2 mmWave Sensor with Home Assistant | Seeed Studio Wiki

MR60BHA2 Heartbeat mmWave Sensor with Home Assistant

https://wiki.seeedstudio.com/ha_with_mr60bha2/

**Comment by coasttech** [comment]
Can this work from more than one person? I have myself and my wife plus two dogs.
